modules/tinymce/src/themes/silver/test/ts/browser/editor/ToolbarBottomTest.ts

```diff
@@ -89,11 +89,11 @@ describe('browser.tinymce.themes.silver.editor.ToolbarBottomTest', () => {
         toolbar: Arr.range(10, Fun.constant('bold | italic ')).join('')
       },
       initial: [{
-        clickOn: 'button[title="More..."]',
+        clickOn: 'button[title="Reveal or hide more toolbar items"]',
```

Comment:
I think this should be "Reveal or hide **additional** toolbar items". This is because the meaning of "hide more" is a bit ambiguous. 

"Additional" indicates that these items are an _extension_ to the existing items already in the unexpanded toolbar. "More" does not clearly indicate this, and can denote the items that are on the unexpanded toolbar as well.